docs/reference/eql/functions.asciidoc

```diff
@@ -8,8 +8,71 @@ experimental::[]
 
 {es} supports the following EQL functions:
 
+* <<eql-fn-startswith>>
 * <<eql-fn-substring>>
 
+[discrete]
+[[eql-fn-startswith]]
+=== `startsWith`
+
+Returns `true` if a source string begins with a provided substring.
```

Comment:
One important bit to mention is that the function is case insensitive. [Not sure if it will always be like this](https://github.com/elastic/elasticsearch/issues/54411) but, for the moment, it's ignoring case sensitivity.

Reply:
Good catch @astefan. I noted this and added an example with b7d36cac677.